docs/website/README.md

```diff
@@ -1,38 +1,63 @@
 # Website
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator. The actual content resides in the `docs` folder.
+The website is a Node.js application.
 
+The documentation is generated using [Docusaurus 2](https://docusaurus.io/), a modern static website generator. 
+Docusaurus consumes content from the `./docs` folder (at `./docs/website/docs` in this repo). The content includes:
```

Comment:
Also Docusaurus consumes content from the `./blog` (`./docs/website/blog`), please mention this in the doc.
here is a generated blog: https://dlthub.com/docs/blog

Reply:
Thanks for adding that.   Appended a line about blogs.  It would be a bigger rewrite to generalise docs and blogs as 'website content`, so for now a shovel remains a shovel, not a digging implement.